### PR TITLE
Add support for emulating edge runtime

### PR DESCRIPTION
**Pull request by codetheweb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,8 @@
     "typescript": "^4.5.4"
   },
   "dependencies": {
+    "edge-runtime": "2.5.4",
+    "esbuild-plugin-environment": "0.2.4",
```
Comment (codetheweb) [inline package.json:74]: `tsup` already transitively required esbuild

File: README.md [hunk 2]
```diff
@@ -66,6 +66,15 @@ We then construct a main export file that knows how to parse `next.config.js`
 and route to the correct files, which we've statically analyzed and are included
 in the generated `dist/index.js` file.
 
+## Caveats
+
+Vercel's Edge Runtime is supported only when running in a Node.js environment. In other words, a bundled nsm project cannot be run in the Edge Runtime.
```
Comment (seveibar) [inline README.md:71]: Hmm this isnt a fundamental limitation right? We could use eg an edge fake inside seam connect and run it on edge?

**Review by seveibar — APPROVED** [review]
Awesome!

File: README.md (hunk 2 above)
Comment (codetheweb) [reply README.md:71]: I don't think it's a fundamental limitation, but just wanted to limit scope in this initial PR

Comment (seveibar) [reply README.md:71]: yep i agree

**Pull request merged**